nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/pom.xml

```diff
@@ -197,6 +197,21 @@
         <dependency>
             <groupId>org.springframework.security.extensions</groupId>
             <artifactId>spring-security-saml2-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.santuario</groupId>
+                    <artifactId>xmlsec</artifactId>
+                </exclusion>
```

Comment:
Is it necessary to exclude the dependency here? It should be sufficient to either include the explicit version, or use the dependency management block in the parent Maven configuration to set the version.